_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -842,130 +857,683 @@ If you need to brush up on how the IaC Library works, read the
+  description = "A feature flag to enable or disable this module."
+  type        = bool
+  default     = false
+// TODO -> this needs to be true, as IAM Access Analyzer needs to be enabled for CIS across all regions. Maybe we should even remove this variable from the example?
+}
+
+# This variable sets the type of the IAM Access Analyzer. Depending on the trust zone required for the analyzer to cover,
+# the possible values are:
+# - ACCOUNT;
+# - ORGANIZATION.
+# When set to ACCOUNT - the trust zone for the IAM Access Analyzer will be limited to just the account and resources
+# only within this account will be scanned, instead of the organizational boundaries and its policies.
+# When set to ORGANIZATION - the trust zone will be set to the whole organization, which allows the IAM Access Analyzer
+# to scan relevant resources shared and applicable within the bounds of the organization. The AWS account with type of
+# IAM Access Analyzer type set to ORGANIZATION, must
+# - an AWS organization master account;
+# - or be part of an AWS organization & set as a delegated admin for this feature.
+#
+# For more information, please read here: https://docs.aws.amazon.com/IAM/latest/UserGuide/what-is-access-analyzer.html
+variable "iam_access_analyzer_type" {
+  description = "If set to ORGANIZATION, the analyzer will be scanning the current organization and any policies that refer to linked resources and their policies within the organization's boundaries."
+  type        = string
+  default     = "ORGANIZATION"
+}
+
+variable "iam_access_analyzer_name" {
+  description = "The name of the IAM Access Analyzer module"
+  type        = string
+  default     = "baseline_root-iam_access_analyzer"
 }
 ----
 
-With the module in place, it's time to test your code. See
-link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library#manual_tests_terraform[Manual tests for
-Terraform code] and
-link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library#automated_tests_terraform[Automated tests
-for Terraform code] for instructions.
+Finally, add some useful outputs in `outputs.tf`:
+
+.infrastructure-modules/landingzone/account-baseline-root/outputs.tf
+[source,hcl]
+----
+# ---------------------------------------------------------------------------------------------------------------------
+# CONFIG OUTPUTS
+# ---------------------------------------------------------------------------------------------------------------------
+
+//TODO -> this needs fixing in the example - currently not referencing a real output
+//output "config_s3_bucket_name" {
+//  description = "The name of the S3 bucket used by AWS Config to store configuration items."
+//  value       = module.root_baseline.config_s3_bucket_name
+//}
+//
+//output "config_s3_bucket_arn" {
+//  description = "The ARN of the S3 bucket used by AWS Config to store configuration items."
+//  value       = module.root_baseline.config_s3_bucket_arn
+//}
```

Comment:
These are currently not recognised & thus is invalid tf. This has been taken from `cis-repo/examples/landingzone/account-baseline-root`

Reply:
Ok, we are probably still waiting to update this example.

Reply:
Yep, I thought so. I'll leave the related comments in this PR to check back on this.